scripts/categorizer.py

```diff
@@ -385,14 +406,45 @@ def main():
         is_conditional = determine_if_conditional(line_number, multiline_number, line, categorizations) # This might be an exclusive one
         is_function_def = determine_if_function_def(line_number, multiline_number, line, categorizations)
         is_assignment = determine_if_assignment(line_number, multiline_number, line, categorizations, is_function_def)
+        is_function_call = determine_if_function_call(line_number, multiline_number, line, categorizations)
 
         categorizations.append(LineAndCats(line=line, multiline_statement_number=multilines[line_number], comment=0,
-                                           conditional=is_conditional, empty=0, func_def=is_function_def, assignment=is_assignment))
+                                           conditional=is_conditional, empty=0, func_def=is_function_def, assignment=is_assignment, func_call= is_function_call))
+
+        categorizations_numbers = range(0, 8)
+        hash_string = "%s" % line
```

Comment:
Write a sentence on what you are doing here then make a comment then use that comment to maybe rewrite the code..